File: web-app/online-warehouse/src/app/carrier/dto/carrier-list.dto.ts
class CarrierListDto {
  id: number;
  name: string;
  taxNumber: string;
  carrierType: string;
  trusted: boolean;

  constructor(id: number, name: string, taxNumber: string, carrierType: string, trusted: boolean) {
    this.id = id;
    this.name = name;
    this.taxNumber = taxNumber;
    this.carrierType = carrierType;
    this.trusted = trusted;
  }
}

File: core/src/main/java/by/itechart/carrier/dto/CarrierListDto.java
package by.itechart.carrier.dto;

import by.itechart.carrier.enums.CarrierType;
import lombok.Data;

@Data
public class CarrierListDto {
    private Long id;
    private String name;
    private String taxNumber;
    private CarrierType carrierType;
    private Boolean trusted;
}
